filebeat/module/haproxy/http/test/haproxy.log

```diff
@@ -0,0 +1 @@
+Jul 30 09:03:52 localhost haproxy[32450]: 1.2.3.4:38862 [30/Jul/2018:09:03:52.726] incoming~ docs_microservice/docs 0/0/1/0/2 304 168 - - ---- 6/6/0/0/0 0/0 {docs.example.internal||} {|||} "GET /component---src-pages-index-js-4b15624544f97cf0bb8f.js HTTP/1.1"
```

Comment:
This file may need a new line at the end to ensure that this is read on tests.

Reply:
Pushed the newline. No change though

Reply:
Ah sorry, been trying to run the tests locally but the environment is not cooperating with me. (I get different failures from the other unrelated tests too)